controllers/fenceagentsremediation_controller.go

```diff
@@ -221,10 +230,22 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 	}
 	if meta.IsStatusConditionTrue(far.Status.Conditions, v1alpha1.FenceAgentActionSucceededType) &&
 		!meta.IsStatusConditionTrue(far.Status.Conditions, commonConditions.SucceededType) {
-		// Fence agent action succeeded, and now we try to remove workloads (pods and their volume attachments)
-		r.Log.Info("Manual workload deletion", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
-		if err := utils.DeleteResources(ctx, r.Client, req.Name); err != nil {
-			r.Log.Error(err, "Manual workload deletion has failed", "CR's Name", req.Name)
+		if far.Spec.RemediationStrategy == v1alpha1.ResourceDeletionRemediationStrategy {
+			// Fence agent action succeeded, and now we try to remove workloads (pods and their volume attachments)
+			r.Log.Info("Manual workload deletion", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
```

Comment:
It might be best to change the term `Manual workload deletion` to `Resource deletion - manually deleting workload`, since there are `OutOfServiceTaint` and `ResourceDeletion` remediation types